snapshot/index.html

```diff
@@ -1859,15 +1859,18 @@ <h5 id="x4-2-4-1-localizable-strings"><bdi class="secno">4.2.4.1 </bdi>Localizab
 								<span>Note</span>
 							</div>
 							<div class="">
-								<p> If the base direction value were not set in the last example, the text would be
+								<p>If the base direction value were not set in the last example, the text would be
 									displayed, following the Unicode Bidirectional Algorithm&#160;[<cite><a
 											class="bibref" data-link-type="biblio" href="#bib-bidi"
 											title="Unicode Bidirectional Algorithm">bidi</a></cite>] and due to the
 									presence of a Latin character starting the string, as: </p>
 								<p dir="ltr">HTML היא שפת סימון.</p>
-								<p> However, that would be incorrect. The extra <code>direction</code> value is
-									necessary to control the display to yield: </p>
+								<p>However, that would be incorrect. The extra <code>direction</code> value is necessary
+									to control the display to yield: </p>
 								<p dir="rtl">HTML היא שפת סימון.</p>
+								<p>(Note that the <var>value</var> field in the example represents the text as it is
+									stored in memory, hence the discrepancy between it and the two renderings depicted
+									here.)</p>
```

Comment:
I may also add something like that in the note: "Text editors may also display the text using the Unicode Bidirectional Algorithm only, instead of the memory order."

Reply:
@iherman I'm not sure what you're trying to say here. When you say 'the text', which text are you referring to?  Do you mean that if you try to cut and past the text of the example you will lose the bdo ordering? Note btw that the first example in the note does exactly what you seem to be describing here.

Reply:
Here's how I've reworded the paragraph:

> Note that the value field in the example represents the text as it is stored in memory, hence the discrepancy between it and the two renderings depicted here. Text editors might also display the JSON value differently (e.g., using the Unicode Bidirectional Algorithm only).

Reply:
If I cut-and-paste the right text (ie, the one with the HTML on the right hand side and properly display the Hebrew text), and then put it into a json file in my screen editor, what I get is the first example in the note and not the memory order as in our example...